### ISSUE TITLE
is it possible to add m2m change signal handler

### ISSUE THREAD

**Issue by chunkai-meng** [MrThearMan/django-signal-webhooks#27]
### I have read the docs thoroughly before making this feature request.

- [X] Yes

### I have read through other open issues, and my issue is not a duplicate.

- [X] Yes

### Description

Hi @MrThearMan,

Thank you for your contributions.

When a model's m2m set changes, we need to send a signal to the webhook to notify the clients
I can't find a way to make it in the current settings.

is it possible to add an m2m handler like:

```py
@receiver(signals.m2m_changed, dispatch_uid=webhook_settings.DISPATCH_UID_POST_M2M_CHANGED)
def webhook_m2m_handler(sender: ModelBase, **kwargs) -> None:
    kwargs: PostM2MData
    webhook_handler(instance=kwargs["instance"], method="M2M_CHANGE")
```

### Motivation

To send data to webhooks when m2m changed

### Would you like to solve this issue yourself with a pull request?

Yes

**Comment by MrThearMan** [comment]
There isn't a way to do this currently, so pull requests are welcome!

**Comment by MrThearMan** [comment]
Hey, I had some free time today, so I decided to implement this. Released on [0.3.0](https://github.com/MrThearMan/django-signal-webhooks/releases/tag/v0.3.0), let me know if it works for you!
